src/Common/src/Interop/Windows/Crypt32/Interop.certificates_imports.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
FIXED: I removed the "_imports" suffix for the *.CS files that just contain DLL Imports.